AirCasting/SearchAndFollow/SearchLocation/SearchViewModel.swift

```diff
@@ -14,26 +14,69 @@ class SearchViewModel: ObservableObject {
     @Published var addressName = ""
     @Published var addresslocation = CLLocationCoordinate2D(latitude: 20.0, longitude: 20.0)
     @Published var alert: AlertInfo?
-    @Published private var parameter: MapDownloaderMeasurementType = .ozone
-    @Published private var sensor: MapDownloaderSensorType = .OzoneSensor
+    @Published var measurementTypes = [SearchParameter]()
+    @Published var sensorTypes = [SearchParameter]()
+    
+    init() {
```

Comment:
move init between properties and functions (this is the most followed convention)